lib/rubocop/cop/generator.rb

```diff
@@ -98,8 +98,9 @@ def on_send(node)
         end
       SPEC
 
-      def initialize(name, output: $stdout)
+      def initialize(name, github_user, output: $stdout)
```

Comment:
Perhaps this should also be a keyword argument with default `"your_id"`?

Reply:
Since I specified `github_user` argument explicitly in production code, then I did not use default keyword argument.

Also I was happy to meet you too in Taiwan 😃